data/fedora-update.yaml

```diff
@@ -449,18 +455,73 @@ python-django-dpaste:
       Project is retired, replaced by not-yet-packaged `dpaste`
     links:
       homepage: https://pypi.python.org/pypi/django-dpaste/0.2.4
+python-django-dynamite:
+    dead-upstream: true
+    links:
+      homepage: https://github.com/z4r/django-dynamite
+    note: |
+      Not maintained since 2009
```

Comment:
Sure. The problem is that the year seems off on lots of packages in this PR. So I'm curious how you decide on the years.